### ISSUE TITLE
Emaggable drones

### ISSUE THREAD

**Issue by ShadowDeath6** [yogstation13/yogstation-classic#338]
Title

Reverses law 3 upon emag, maybe have a high chance of the emag failing on the drone so it can bail out. 

Law 3- Your goals are to build, maintain, repair, improve, and power to the best of your abilities, You must never actively work against these goals.

Emagged law 3- Your goals are to build, maintain, repair, improve, and power to the best of your abilities, You must always actively work against these goals.

Basically makes the drone fuck around and sabotage the station, but it obviously can't interfere with living things so no stupid shit like releasing singlo or shocking doors. 

Incoming shitstorm :^)

**Comment by ParagonPrime** [comment]
Me like

me very like

**Comment by X-TheDark** [comment]
Please no, I'm against legalizing swarmers.

**Comment by ShadowDeath6** [comment]
They need an obvious visual tell because if they didn't then they would be actual satan to deal with.

**Comment by ghost** [comment]
They can flash red similar to beep sky.

**Comment by Kmc2000** [comment]
Me likey

Set the drone colour to red and have it twitch violently due to the fact that the emag would break it's tiny little motors

**Comment by ShadowDeath6** [comment]
It shouldn't be super obvious, maybe just random _beeps ominously_ messages.  Red color works well though

**Comment by shrimptk** [comment]
here comes the meta train

**Comment by X-TheDark** [comment]
Also, let's not forget, these bastards are impossible to deal with without explosives/AoE weapons as they pass through doors/tables/etc and you need to click on them for a shot to hit them.

**Comment by ghost** [comment]
swarmers exist for a reason

**Comment by X-TheDark** [comment]
Yup, instead of making drones emagable, we should probably make Swarmer an actual random event that happens.

**Comment by Alblaka** [comment]
1. Emagg drones to turn them into swarmers
2. Kek
3. Profit

**Comment by ShadowDeath6** [comment]
what was I thinking
